packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -44,6 +34,16 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
     // @dev the block the contract was deployed at
     uint256 public immutable deployBlock;
 
+    function bridgeStatuses(bytes32 transactionId) public view returns (BridgeStatus status) {
+        return bridgeTxDetails[transactionId].status;
+    }
+
+    function bridgeProofs(bytes32 transactionId) public view returns (uint96 timestamp, address relayer) {
```

Comment:
Let's leave the constructor as a first defined function, moving these all the way below would be canonical.

Reply:
NP. if cool w/ you, i'll rearrange this and most of the other components of FastBridgeV2 in a separate PR after this merges that focuses on eliminating all solhint warnings

Reply:
Sounds good